.github/workflows/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+    attr_accessor :title, :author, :rentals
+  
+    def initialize(title, author)
+      @title = title
+      @author = author
+      @rentals = []
+    end
+  
+    def add_rental(date, person)
+      rental = Rental.new(date, person, self)
+      @rentals << rental
+      person.add_rental(rental)
```

Comment:
- [ ] The Person class should have an independent `add_rental` method like the Book class for creating the association between both classes. So writing this method this way is not a good idea. Kindly remove the second and third lines in this method. We would refactor the Rental class to add itself to the `@rentals` of Book and Person, along with the Person class. For now, kindly change this method to this:

  ```
  def add_rental(date, person)
    Rental.new(date, person, self)
  end
  ```